src/styles/TextFieldStyle.qml

```diff
@@ -25,8 +25,9 @@ TextFieldStyle {
     }
 
     font {
+        id:myFont
         family: echoMode == TextInput.Password ? "Default" : "Roboto"
-        pixelSize: 16 * Units.dp
+        pointSize: 16 //* Units.dp
```

Comment:
if you're sure to use pointSize instead of Unit.dp you can delete this comment // *Unit.dp